deploy/manifests/00-crds.yaml

```diff
@@ -3703,6 +3703,40 @@ spec:
                       - roleId
                       - secretRef
                       type: object
+                    kubernetes:
+                      description: This contains a Role and Secret with a ServiceAccount
+                        token to authenticate with vault.
+                      properties:
+                        mountPath:
+                          description: The kubernetes mount path to be used when authenticating
+                            (/v1/auth/%s/login). If unspecified, the default value
```

Comment:
Using %s here feels awkward.. is there a better way we can communicate what we mean? Maybe just be more explicit ```The value here will be used as part of the path used when authenticating with vault, for example if you set a value of "foo", the path used will be `/v1/auth/foo/login`. If unspecified, ...```